core/src/libraries/BalanceSheet.sol

```diff
@@ -76,17 +70,21 @@ library BalanceSheet {
 
     function computeProbePrices(
         uint56 metric,
-        uint160 sqrtMeanPriceX96,
+        uint256 sqrtMeanPriceX96,
```

Comment:
Still expect it to be uint160 coming in, this is just to avoid annoying typecasting inside the fn.